internal/engine/experiment/dnscheck/dnscheck.go

```diff
@@ -120,10 +120,11 @@ var (
 )
 
 // Run implements model.ExperimentSession.Run
-func (m *Measurer) Run(
-	ctx context.Context, sess model.ExperimentSession,
-	measurement *model.Measurement, callbacks model.ExperimentCallbacks,
-) error {
+func (m *Measurer) Run(ctx context.Context, args *model.ExperimentArgs) error {
+	_ = args.Callbacks
```

Comment:
no need to assign here no?

Reply:
No, I left it for clarity to highlight we're not using the callbacks. We may need to use them and the `_` will tell us so.

Reply:
I mean: previously we were passing a `callbacks` argument and ignoring and I want to keep track of that.